legend-engine-pure-code-compiled-core/src/main/resources/core/pure/mapping/mappingExtension.pure

```diff
@@ -317,6 +317,16 @@ function  meta::pure::mapping::from(tds:TabularDataSet[1], mapping:Mapping[1], r
    $tds;
 }
 
+function  meta::pure::mapping::from<T|m>(t:T[m], executionParameters: meta::legend::service::metamodel::SingleExecutionParameters[1]):T[m]
+{
+   $t
+}
+
+function  meta::pure::mapping::from(tds:TabularDataSet[1], executionParameters: meta::legend::service::metamodel::SingleExecutionParameters[1]):TabularDataSet[1]
```

Comment:
Do we need this function?

The above should handle both

Reply:
with from function I saw this pattern, so added it, but if not required would delete